build/dappnode/scripts/dappnode_uninstall.sh

```diff
@@ -20,6 +20,9 @@ uninstall() {
     # Remove containers, volumes and images
     docker-compose $DNCORE_YMLS down --rmi 'all' -v
 
+    # Remove dncore_network
+    docker network remove dncore_network
```

Comment:
Network may not exist, guard against that

Reply:
Since we're not using `set -e` for this script, in the case that this commands fail it should throw an error and continue the normal execution, but if you want it could be rewritten like this for instance:
```
docker network inspect dncore_network && docker network remove dncore_network
```
Personally, I don't see any benefit, unless we plan to add the `set -e` in the future

P.D: `-e  Exit immediately if a command exits with a non-zero status.`